src/Raven.Server/Documents/Subscriptions/SubscriptionStorage.cs

```diff
@@ -690,6 +680,9 @@ public void HandleDatabaseRecordChange(DatabaseRecord databaseRecord)
                         continue;
                     }
 
+                    if (subscriptionStateKvp.Value.IsSubscriptionActive() == false)
```

Comment:
can you explain this as well?

Reply:
in code below we check if we need to cancel the `SubscriptionConnection`, in case `IsSubscriptionActive` is false, we don't have `SubscriptionConnection` so those checks are not necessary, in case we continue we will hit `if (whoseTaskIsIt != _serverStore.NodeTag)` and spam the log (as was reported in the issue)